### PR TITLE
changes the configuration to use a polynomial to define the mapping o…

### PR DESCRIPTION
**Pull request by colinwinslow**
…f voltage readings onto telemetry units, and adds documentaion.

### CODE REVIEW

File: AdamSensors/v1/2020_homeoffice_pandemic_test.yaml [hunk 1]
```diff
@@ -1,20 +1,31 @@
...
+analog_input_1_coefficients: [1, 0]
 analog_input_2_type: None
-analog_input_2_min: -10.
-analog_input_2_max: 10.
-analog_input_3_type: None
-analog_input_3_min: -10.
-analog_input_3_max: 10.
+analog_input_2_coefficients: [1, 0]
+analog_input_3_type: Pressure
+analog_input_3_coefficients: [344738, 0]
```
Comment (patrickingraham) [inline AdamSensors/v1/2020_homeoffice_pandemic_test.yaml]: Do you care about whether things are floats/ints ?
Does this have a type in the schema? Probably worth matching that if so..

File: AdamSensors/v1/_labels.yaml [hunk 2]
```diff
@@ -1,2 +1,2 @@
 # Labels for recommended settings; a dict of label: config_file
-2020_homeoffice_pandemic_test: 2020_homeoffice_pandemic_test.yaml
+test: homeoffice_pandemic_test.yaml
```
Comment (patrickingraham) [inline AdamSensors/v1/_labels.yaml]: how about `initial_test_config.yaml`

**Review by patrickingraham — CHANGES_REQUESTED** [review]

File: AdamSensors/v1/_labels.yaml (hunk 2 above)
Comment (tribeiro) [reply AdamSensors/v1/_labels.yaml]: @colinwinslow , can you update the labels file with the new file name?

File: AdamSensors/v1/homeoffice_pandemic_test.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# The AdamSensors CSC reads voltages off of the ADAM 6024's six analog input channels, in
```
Comment (tribeiro) [inline AdamSensors/v1/homeoffice_pandemic_test.yaml]: Please, rename this file to something like `initial_test_config.yaml`, as suggested above.

**Review by tribeiro — CHANGES_REQUESTED** [review]
Please, rename the configuration and update the labels file.

**Review by tribeiro — APPROVED** [review]

**Review by patrickingraham — APPROVED** [review]
Changes addressed.

**Pull request merged**